docs/internal/Element-Type-Specs.md

```diff
@@ -266,9 +266,13 @@ _Multi-choice question, with one allowed option, displayed as Drop-down list (Co
 - **search**: `boolean` (default: `false`) -- if `true`, the list of options can be searched and filtered by user
 - **optionsDisplayProperty**: If `options` (above) consists of an array of objects, this parameter specifies the field of each object to be displayed in the options list. For example, if `options` was a list of organisation objects (i.e. `{orgId, name, licenceNumber}`), you'd probably specify `name` as the `optionsDisplayProperty`. Note that even though one field is displayed to the user in the Dropdown list, the _entire_ selected object is saved as the selection. And if `optionsDisplayProperty` refers to a field that doesn't exist on the supplied object, the plugin will fail and show in error in the application.
 - **hasOther**: `boolean` (default `false`) -- if `true`, allows the user to enter a custom "free text" value instead of one of the pre-defined options.
+- **multiSelect**: `boolean` -- whether or not the user can select multiple items for their response (default: `false`)
+
+Note: if including an "Other" options, then options should only be an array of strings, or else you'll be mixing objects with strings, which will cause problems. In other words, you should never have both optionsDisplayProperty and hasOther defined in the same question element.
```

Comment:
It seems to work on initial testing with both hasOther and multiSelect for an array of objects.

Maybe it would be nice if it failed more loudly if this would cause issues.

Reply:
You can use `hasOther` with `multiSelect` just fine -- you just can't use `optionsDisplayProperty` with `hasOther`. This has always been the case, it just wasn't documented.